components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -45,8 +45,8 @@ public static class ExitCode {
     @Option(names = {"-m", "--mapping-file"}, description = "Specifies a mapping file that configures the keys used to fetch important custom values.")
```

Comment:
Do we need the `-m` option here, or is the `--mapping-file` option sufficient? I assume we always call this cmd line tool with a fixed arg name.

Reply:
We are currently using `-m`, but it's easy enough to change it, and I think that makes sense.

✅ Done